WSEvents.cpp

```diff
@@ -672,34 +699,35 @@ void WSEvents::Heartbeat() {
         }
 
         if (Heartbeat_Streaming_active) {
-            if (streaming_active) obs_data_set_bool(data, "streaming", streaming_active);
-            else obs_data_set_bool(data, "streaming", false);
+            obs_data_set_bool(data, "streaming", streaming_active);
         }
-        if (Heartbeat_TotalStreamTime_active) {
-            if (streaming_active) {
+        if (Heartbeat_TotalStreamTime_active && streaming_active) {
                 uint64_t totalStreamTime = (os_gettime_ns() - _stream_starttime) / 1000000000;
                 obs_data_set_int(data, "total-stream-time", totalStreamTime);
-            }else{
-                obs_data_set_int(data, "total-stream-time", 0);
-            }
         }
+		if (Heartbeat_TotalStreamBytes_active) {
```

Comment:
Incorrect indentation here too. Same proposition as above if you want.

Reply:
Untabified all.